users/templates/base.html

```diff
@@ -35,7 +35,7 @@
     {% block extra_head %}{% endblock %}
 </head>
 <body>
-{% if user.is_authenticated %}
+{% if user.is_authenticated and not user.need_change_password %}
```

Comment:
We will handle this differently so please delete it. We need to add middleware to check permissions to access site :) Task for this is here https://trello.com/c/n2fAgd9V/33-add-middleware-for-permission-checking

Reply:
Ok.